extensions/vscode/webviews/homeView/src/components/tree/TreeItem.vue

```diff
@@ -159,13 +164,24 @@ const toggleExpanded = () => {
         line-height: 22px;
         color: inherit;
         white-space: pre;
+        min-width: 0;
+        overflow: hidden;
+        text-overflow: ellipsis;
       }
 
       .tree-item-description {
+        line-height: 22px;
         font-size: 0.9em;
         margin-left: 0.5em;
-        opacity: 0.7;
```

Comment:
Since `description` can contain an `input` now we don't want the description to _always_ be `opacity: 0.7`. Now we apply this only if the description doesn't contain an `input` (see below)